src/app/components/client/DataBrokerProfiles.tsx

```diff
@@ -61,14 +68,22 @@ export type DataBrokerProfileCardProps = {
 
 export const DataBrokerProfileCard = (props: DataBrokerProfileCardProps) => {
   const l10n = useL10n();
+  const recordTelemetry = useTelemetry();
 
   return (
     <div className={styles.dataBrokerProfileCard}>
       <span className={styles.dataBrokerName}>
         {getDataBrokerName(props.data.data_broker)}
       </span>
-
-      <a href={props.data.link} target="_blank">
+      <a
+        href={props.data.link}
+        target="_blank"
+        onClick={() => {
+          recordTelemetry("link", "click", {
+            link_id: `viewed_${props.data.data_broker}`,
```

Comment:
Maybe we should keep `*_data_broker` to make it easier to filter for?
```suggestion
            link_id: `viewed_data_broker_${props.data.data_broker}`,
```